components/combobox/apiExamples/externalSelection.html

```diff
@@ -0,0 +1,21 @@
+<p style="color: white;">
+    Type 'a' in the combobox for best testing results :)
+</p>
+
+<auro-combobox id="externalSelectionExample" layout="classic" shape="classic" size="lg" placeholder="Placeholder content" required style="width: 249px;">
```

Comment:
Feels like we should be using the `onDark` combobox for this example

Reply:
fixed in next commit